cumulusci/tasks/robotframework/libdoc.py

```diff
@@ -0,0 +1,106 @@
+import os
+import os.path
+import time
+import jinja2
+import robot.utils
+
+import cumulusci
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.tasks import BaseTask
+from cumulusci.core.utils import process_list_arg
+
+from robot.libdocpkg import DocumentationBuilder
+from robot.libraries.BuiltIn import RobotNotRunningError
+
+
+class RobotLibDoc(BaseTask):
+    task_options = {
+        "path": {
+            "description": "The path to the robot library to be documented.  Can be a python file or a .robot file.",
+            "required": True,
+        },
+        "output": {
+            "description": "The output file where the documentation will be written",
+            "required": True,
+        },
+        "title": {
+            "description": "A string to use as the title of the generated output",
+            "required": False,
+        },
+    }
+
+    def _validate_options(self):
+        super(RobotLibDoc, self)._validate_options()
+
+        self.options["path"] = process_list_arg(self.options["path"])
+
+        bad_files = []
+        for input_file in self.options["path"]:
+            if not os.path.exists(input_file):
+                bad_files.append(input_file)
+
+        if bad_files:
+            if len(bad_files) == 1:
+                error_message = "Unable to find the input file '{}'".format(
+                    bad_files[0]
+                )
+            else:
+                files = ", ".join(["'{}'".format(filename) for filename in bad_files])
+                error_message = "Unable to find the following input files: {}".format(
+                    files
+                )
+            raise TaskOptionsError(error_message)
+
+    def _run_task(self):
+        libraries = []
+        processed_files = {}
+        for input_file in sorted(self.options["path"]):
```

Comment:
Why sorted? Seems like it could be nice to keep it in the order specified in the task option, in case there's a primary library that doesn't come first alphabetically.

Reply:
I don't have a good answer. I debated that, and do see value in letting the order be specified in the yml. I see your point, so I've removed the sorting.

Thanks for catching this!